pkg/asset/installconfig/ovirt/cluster.go

```diff
@@ -0,0 +1,52 @@
+package ovirt
+
+import (
+	"fmt"
+	"sort"
+
+	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+func askCluster(c *ovirtsdk4.Connection, p *ovirt.Platform) (string, error) {
+	var clusterName string
+	var clusterByNames = make(map[string]*ovirtsdk4.Cluster)
+	var clusterNames []string
+	systemService := c.SystemService()
+	response, err := systemService.ClustersService().List().Send()
+	if err != nil {
+		return "", err
+	}
+	clusters, ok := response.Clusters()
+	if !ok {
+		return "", fmt.Errorf("there are no available clusters")
+	}
+
+	for _, cluster := range clusters.Slice() {
+		clusterByNames[cluster.MustName()] = cluster
+		clusterNames = append(clusterNames, cluster.MustName())
+	}
+	err = survey.AskOne(&survey.Select{
+		Message: "Pick the oVirt cluster",
+		Help:    "The oVirt cluster under which the VMs will be created.",
```

Comment:
The oVirt cluster where the VMs will be created.

Reply:
done